src/messages/check_offers.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from messages.base import BaseMessage
+
+
+@dataclass
+class CheckOffers(BaseMessage):
```

Comment:
What do you think about adding short comments what the messages are for? Also we don't need to do that now, but maybe later on?